setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+setup(
+    name="linked_list",
```

Comment:
This repository will be for more than just the Linked List. It'll be for all of your data structures and algorithms